views/js/runner/plugins/navigation/previous.js

```diff
@@ -66,15 +79,46 @@ define([
             }));
 
             //attach behavior
-            this.$element.on('click', function(e){
-                e.preventDefault();
-                if(self.getState('enabled') !== false){
-                    self.disable();
+            function doPrevious(previousItemWarning) {
+                var context = testRunner.getTestContext();
 
-                    testRunner.previous();
+                previousItemWarning = previousItemWarning || false;
```

Comment:
Is it useful ? If the `previousItemWarning` variable is already a falsey, what is the purpose here ? type enforcement optimisation ?